### PR TITLE
Features/people list

### PR DESCRIPTION
**Pull request by ursmal1948**
PEOPLE LIST implemented in movie browser. Small changes in responsiveness still to be done

### CODE REVIEW

File: src/common/MainContainer/styled.js [hunk 1]
```diff
@@ -71,6 +71,12 @@ export const TilesContainer = styled.div`
   justify-items: center;
   width: 1372px;
 
+  ${({ peopleListFlag }) =>
+    peopleListFlag &&
+    css`
+      grid-template-columns: repeat(6, 1fr);
```
Comment (krzysztof-gronski) [inline src/common/MainContainer/styled.js:77]: try to use auto-fill value to adjust width on small screens, for example:  grid-template-columns: repeat(auto-fill, minmax(170px, 1fr));

File: src/common/Tile/styled.js [hunk 2]
```diff
@@ -11,6 +11,15 @@ export const TileContainer = styled.div`
   box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
   border-radius: 5px;
```
Comment (krzysztof-gronski) [inline src/common/Tile/styled.js:13]: i see common styles remains unchanged but movie tile is on movieDetails page is broken, any idea which modification causes that ? 🤔

File: src/common/MainContainer/styled.js (hunk 1 above)
Comment (ursmal1948) [reply src/common/MainContainer/styled.js:77]: Yess, i finally have come to right solution. Anyway i think we will not merge this branch :(

File: src/common/Tile/styled.js (hunk 2 above)
Comment (ursmal1948) [reply src/common/Tile/styled.js:13]: Hmm i could have deleted sth by mistake. But no worries, everything looks as it should in my new branch features/peopleList redesign. WE just need to merge it

**Pull request closed without merging**